pkg/validation/component.go

```diff
@@ -135,6 +129,13 @@ func (c *component) GetUser(ctx context.Context, realmName string, userID string
 }
 
 func (c *component) UpdateUser(ctx context.Context, realmName string, userID string, user api.UserRepresentation) error {
+	var validationCtx = validationContext{
+		ctx:       ctx,
```

Comment:
I feel quite uncomfortable with such architecture ...
component provided inside another struct doesn't follow our currentway of doing things until now. Do we really need to perform such thing ? Why the methods of ValidationContext are not just private method of this component ?
Maybe I am missing one of the constraint

Reply:
I removed component from ValidationContext and replaced func (v *ValidationContext) xxx(...) by func (c *component) xxx(v *ValidationContext, ...)